tests/Future.ts

```diff
@@ -23,21 +25,44 @@ describe("Future.of", () => {
         return ensureFailedWithValue(5, Future.of(Promise.reject(5)).toPromise());
     });
 });
-describe("Future basics", () => {
-    it("is lazy", () => {
-        let i=0;
-        Future.ofCallbackApi<number>(done=>{++i; done(i)});
-        assert.deepEqual(0, i);
+describe("Future.ofCallback (these tests will fail on windows)", () => {
+    it("properly operates with fs.readFile", async () => {
+        const passwd = await Future.ofCallback<string>(
+            cb => fs.readFile("/etc/passwd", "utf-8", cb));
```

Comment:
lol random
Probably put something crossplatform (like a file from inside the repo) or your own (spy?) function.